col/src/main/java/vct/col/ast/syntax/PVLSyntax.java

```diff
@@ -50,22 +50,24 @@ public static Syntax get(){
       syntax.addFunction(Old,"\\old");
 
       syntax.addFunction(OptionSome, "Some");
-                                                                 
-      syntax.addFunction(MapBuild, "buildMap");           
-      syntax.addFunction(MapEquality, "equalsMap");       
-      syntax.addFunction(MapDisjoint, "disjointMap");     
-      syntax.addFunction(MapKeySet, "keysMap");           
-      syntax.addFunction(MapCardinality, "cardMap");      
-      syntax.addFunction(MapValueSet, "valuesMap");       
-      syntax.addFunction(MapGetByKey, "getFromMap");      
-      syntax.addFunction(MapRemoveKey, "removeFromMap"); 
+      syntax.addFunction(OptionGet, "getOption");
+      syntax.addFunction(OptionGetOrElse, "getOrElseOption");
+
+      syntax.addFunction(MapBuild, "buildMap");
+      syntax.addFunction(MapEquality, "equalsMap");
+      syntax.addFunction(MapDisjoint, "disjointMap");
+      syntax.addFunction(MapKeySet, "keysMap");
+      syntax.addFunction(MapCardinality, "cardMap");
+      syntax.addFunction(MapValueSet, "valuesMap");
+      syntax.addFunction(MapGetByKey, "getFromMap");
+      syntax.addFunction(MapRemoveKey, "removeFromMap");
       syntax.addFunction(MapItemSet, "itemsMap");
 
-
       syntax.addFunction(TupleFst, "getFst");
       syntax.addFunction(TupleSnd, "getSnd");
 
```

Comment:
why three new lines here?

Reply:
Does not seem to have a reason, will be fixed in next commit.